exonum/src/events/noise/tests.rs

```diff
@@ -107,6 +112,84 @@ fn test_converted_keys_handshake() {
         .expect("Unable to transition session into transport mode");
 }
 
+#[test]
+fn test_noise_encrypt_decrypt_short_message() {
+    test_encrypt_decrypt(64);
+}
+
+#[test]
+fn test_noise_encrypt_decrypt_long_message() {
+    test_encrypt_decrypt(MAX_MESSAGE_LENGTH + 1);
+}
+
+#[test]
+fn test_noise_encrypt_decrypt_bogus_message() {
```

Comment:
I wonder if it could make sense to test `MessagesCodec` here.

Reply:
`MessagesCodec` is tested for some wrong input in their own module. I think those tests combined with `encrypt_decrypt` tests introduced here is enough to cover codec functionality. But maybe I miss something.